spec/dom-to-image-more.spec.js

```diff
@@ -543,9 +548,15 @@
                 const controlUrl = getImageDataURL(controlImage(), 'image/png');
 
                 if (imageUrl !== controlUrl) {
-                    console.log(`        image: ${image.src}`);
-                    console.log(`  imageBase64: ${imageUrl}`);
-                    console.log(`controlBase64: ${controlUrl}`);
+                    console.debug(`
```

Comment:
Lets emit DEBUG log messages that make it easier to see what's different... so some nice HTML